### PR TITLE
chore(composer): Run community site via www with Elgg 2.0

### PR DESCRIPTION
**Pull request by ewinslow**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/brettp/bulk-user-admin"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/ewinslow/spam_throttle"
+        }
+    ]
```
Comment (ewinslow) [inline composer.json]: Once these repos support composer properly (i.e. they accept the PRs, tag the release, and publish on packagist), we can remove these custom repos.

**Comment by ewinslow** [comment]
@Elgg/owners @Elgg/contributors Anyone care to give this a once-over?

**Comment by mrclay** [comment]
LG but haven't tested it. The next step is for this on community-test?

**Comment by ewinslow** [comment]
It's already ui

**Comment by mrclay** [comment]
Why do I see the community home page and not the elgg.org page?

**Comment by ewinslow** [comment]
That is a great question. I haven't figured that out yet

**Comment by ewinslow** [comment]
@mrclay Fix is here: https://github.com/Elgg/community_theme/pull/50

**Pull request merged**